mlflow/types/utils.py

```diff
@@ -238,7 +237,7 @@ def __call__(self, x):
             return DataType.string
         else:
             raise MlflowException(
-                "Unable to map 'np.object' type to MLflow DataType. np.object can"
+                "Unable to map 'object' type to MLflow DataType. object can"
                 "be mapped iff all values have identical data type which is one "
```

Comment:
```suggestion
                "be mapped if all values have identical data type which is one "
```

Reply:
`iff` could be intentional: https://github.com/apache/arrow/pull/12221

Reply:
Ah yes... logical operator shorthand (sorry, it's been a long day). Do we want to have something like that in a DS-focused Exception? (I'm just curious now)